### DISCUSSION TITLE
Craft 4 active entries filters

### DISCUSSION THREAD

**Discussion by gregorterrill** [craftcms/cms#10664] [Ideas]
I've been playing around with the Craft 4.0.0-alpha release and for the most part it's looking great, but I have a few general comments about some of the new features, specifically on the Entries screen. One suggestion is to **make active filters more visually prominent**.

Right now, when you have one or more active filters on an element view, the little filters icon in the Search box uses its active state (grey background, white icon). I find this way too subtle and I think content authors are going to totally miss it and wonder why content isn't showing up when it's being filtered out. I think this is especially problematic when you set a filter for one element source, then click on another element source, and then return to the first and the filters are still active (which to be clear, I do think is the right call, I just don't think it's obvious enough that it's happening). I think there are a couple ways this could be improved, like showing the active filters below the search box or showing a text tag next to the filters icon like "2 active filters":

<img width="831" alt="Screen Shot 2022-03-02 at 10 23 18 AM" src="https://user-images.githubusercontent.com/1774747/156391811-d0461f29-081b-4324-bf40-b6608fa60b29.png">

**Comment by brandonkelly** [comment]
Hi, appreciate the feedback, but can you please edit it to only focus on a single suggestion, and create a new discussion for the other?

**Reply by gregorterrill** [reply]
Sure thing! Moved the second item to #10676
